Source/WebCore/rendering/RenderObject.h

```diff
@@ -469,12 +469,13 @@ class RenderObject : public CachedImageClient, public CanMakeCheckedPtr {
     bool isLegacyRenderSVGForeignObject() const { return type() == Type::LegacySVGForeignObject; }
     bool isRenderSVGForeignObject() const { return type() == Type::SVGForeignObject; }
     virtual bool isLegacyRenderSVGResourceContainer() const { return false; }
-    bool isRenderSVGResourceContainer() const { return type() == Type::SVGResourceClipper; }
+    bool isRenderSVGResourceContainer() const { return type() == Type::SVGResourceClipper || type() == Type::SVGResourceMasker; }
     bool isRenderSVGResourceFilter() const { return type() == Type::SVGResourceFilter; }
     bool isLegacyRenderSVGResourceMarker() const { return type() == Type::SVGResourceMarker; }
```

Comment:
We should decide on a one, unique solution. Clipper has LegacySVGResourceClipper / SVGResourceClipper. For marker we only have SVGResourceMarker at present. I'd go for explicit LegacySVGResourceMasker / SVGResourceMasker.